docs/product-user-guide/src/main/asciidoc/drl-rules-maven-create-proc.adoc

```diff
@@ -0,0 +1,254 @@
+    <artifactId>drools-compiler</artifactId>
+    <version>VERSION</version>
+  </dependency>
+  <dependency>
+    <groupId>org.kie</groupId>
+    <artifactId>kie-api</artifactId>
+    <version>VERSION</version>
+  </dependency>
+  <dependency>
+    <groupId>junit</groupId>
+    <artifactId>junit</artifactId>
+    <version>4.11</version>
+    <scope>test</scope>
+  </dependency>
+</dependencies>
+</project>
+----
++
+For the Maven artifact version supported in {PRODUCT}, see Installing {PRODUCT} on premise.
+
+. Use the `testApp` method in `my-app/src/test/java/com/mycompany/app/AppTest.java` to test the rule. The `AppTest.java` file is created by Maven by default.
++
+. In the `AppTest.java` file, add the required `import` statements to import KIE services, a KIE container, and a KIE session. Then load the knowledge base, insert facts, and execute the rule from the `testApp()` method that passes the fact model to the rule.
++
+In the following example, the required imports are listed and a fact model `DroolsTest.java` is created:
++
+[source,java]
+----
+import org.kie.api.KieServices;
+import org.kie.api.runtime.KieContainer;
+import org.kie.api.runtime.KieSession;
+
+public void testApp() {
+
+  // Load the knowledge base:
+  KieServices ks = KieServices.Factory.get();
+  KieContainer kContainer = ks.getKieClasspathContainer();
+  KieSession kSession = kContainer.newKieSession();
+
+  // Set up the fact model:
+  Person p = new Person();
+  p.setWage(12);
+  p.setFirstName("Tom");
+  p.setLastName("Summers");
+  p.setHourlyRate(10);
+
+  // Insert the person into the session:
+  kSession.insert(p);
+
+  // Fire all rules:
+  kSession.fireAllRules();
+}
+----
++
+. After you create and save all DRL assets in your Maven archetype, navigate to the `my-app` directory in the command line and run the following command to build your files:
++
+[source]
+----
+mvn clean install
+----
++
+The first time you run this command, the build process can take more time than usual. After the build completes, the results are displayed in the command line:
++
+[source]
+----
+...
+
+Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.194 sec
+
+Results :
+
+Tests run: 1, Failures: 0, Errors: 0, Skipped: 0
+
+[INFO]
+...
+[INFO] --------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] --------------------------------------------------------------
+[INFO] Total time: 6.393 s
+...
+[INFO] --------------------------------------------------------------
+----
+. Review the build results to ensure that the build ran properly, and address any errors in the files.
+
+*<@Michael (13 of 13): Again, same question. What should the user expect or do now? Are the DRL files integrated with {PRODUCT} via the knowledge base? Can the user see their stuff in {CENTRAL} now? If so, where? Not sure what happens once the user does this and would like to call out what the overall result is of their doing this using Maven.>*
```

Comment:
@manstis, so the user would compile the created project into a KJAR and then add that KJAR as a dependency in an existing project in Central. This would "integrate" the standalone project and expose the data objects etc., correct? And where exactly does the user add the KJAR as a dependency, i.e., which file in the existing project? I'm looking through the Dev Guide and can't determine that.

And I hear what you're saying about being able to deploy it via REST etc. I want to just provide one method of integration, the above dependency method, to keep it simpler. Will get into more in a separate IDE doc, or the like.